guide/sections/part2/wis2node.adoc

```diff
@@ -4,24 +4,33 @@
+
+
+In the case of NCs, the procedure for PR approved decomissioning of a WIS node is the PR (or designate) will notify the WMO Secretariat of the decomissioning of the WIS node. In the case of DCPCs, the sponsor (i.e., Regional Association or WMO Programme) is the Programme Chair or Regional Information Management Chair shall approve the decommissioning and notify the WMO Secretariat of the decomissioning of the WIS node. When possible, 30 days notice should be provided prior to the decomissioning and information on if the data from the node will also be decomissioned or if it is available via another WIS node or method.  The WMO Secretariat will notify the GISC for the country and the Global Service Providers of the date of the WIS node decomissioning.  
+
+NC/DCPCs operators decomissioning a WIS node shall ensure that obligations relating to data sharing within WIS continue to be met after the WIS Node is decommissioned, for example, by migrating these data sharing obligations to another WIS Node. In the case of DCPCs, this may mean the WIS node responsibilities shift from one member to another and in the case these details should be included in the decomissioning notice to the WMO Serectariat.
 
 ===== Registration and removal of a Dataset
 
-This section describes the process used to register a Dataset so that it may be discovered and shared within WIS2. In cases where a WIS Centre no longer wishes to share a Dataset via WIS, it must be removed as per the procedure described here.
+All data shared in WIS2 must include metadata compliant with the metadata requriements described below.  Data providers must be approved by their PR (or designate such as the member's WIS Focal Point) to ensure quality metadata will be provied along with the data.
 
-TODO: to be completed
+An organization ready to publish a new dataset should contact the WMO Secretariat with the approval of their PR (or designate) to The WMO Secretariat.  The WMO Secretariat will contact the GISC for the organizaiton with the new (meta)data.  The GISC will work with a Global Data Catalogue to verify the metadata.  the GDC will publish a report indicating errors and/or potential improvements (based on discovery metadata KPIs). The GISC should work with the data publisher to remedy issues and incporate suggestions for improvement.  
```

Comment:
Is this workflow for each new dataset? I would expect that once a WIS2. node has been approved that this metadata and data workflow would occur without a lot of oversight and regular monitoring should be sufficient.

"An organization ready to publish a new dataset should contact the WMO Secretariat with the approval of their PR (or designate) to The WMO Secretariat. The WMO Secretariat will contact the GISC for the organizaiton with the new (meta)data. The GISC will work with a Global Data Catalogue to verify the metadata."